clash-lib/src/Clash/Primitives/Intel/ClockGen.hs

```diff
@@ -36,11 +51,20 @@ altpllTF = TemplateFunction used valid altpllTemplate
 alteraPllTF :: TemplateFunction
 alteraPllTF = TemplateFunction used valid alteraPllTemplate
  where
-  used         = [1,2,3]
+  used         = [1..20]
```

Comment:
I guess we need `IgnoredArguments`/`UsedArguments` like we do for `BlackBoxHaskell` for `TemplateFunction`s too :-)